src/Native/pkg/runtime.native.System.Security.Cryptography/dir.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)\.., dir.props))\dir.props" />
+
+  <PropertyGroup>
```

Comment:
Would it be better to set this property in all of the 10 projects for each distro?  I can make that change if you'd like.